src/components/navigation-header/components/product-page-content/components/product-icon-text-lockup/index.tsx

```diff
@@ -0,0 +1,27 @@
+import ProductIcon from 'components/product-icon'
+import { isProductSlug } from 'lib/products'
+import s from './product-icon-text-lockup.module.css'
+
+export interface ProductIconTextLockupProps {
+	slug: string
+	name: string
+}
+
+/**
+ * Render a logo-lik
+ * @returns
```

Comment:
```suggestion
 * Render a logo-lik
```
Perhaps a typo? 😆

Reply:
Indeed, thanks for catching this 💖  Updated to `Render a logo-like lockup of a product icon, plus the product name.`